core/views.py

```diff
@@ -45,69 +59,87 @@ def home(request):
     return render(request, "home.html", context)
 
 
-@ratelimit(key="user_or_ip", rate="1/m", block=True)
+# @ratelimit(key="user_or_ip", rate="1/m", block=True)
 def addHackathons(request):
     if request.method == "POST":
         form = HackathonForm(request.POST)
         if form.is_valid():
+            print("sending the form")
+            form.save(commit=False)
+            form.review_status = ReviewStatus.Pending
+            form.created_by = Hacker.objects.get(id=request.user.id)
+            form.save()
             return redirect("home")
+        else:
```

Comment:
add popups telling the user if the form was submited successfully or rejected

Reply:
see form.errors https://docs.djangoproject.com/en/5.1/ref/forms/api/#django.forms.Form.errors

Reply:
this was not changed